examples/errors/1-Process-Request-Send-400_INVALID_CHECKSUM.json

```diff
@@ -358,40 +352,6 @@
         ]
       }
     },
-    {
-      "fullUrl": "urn:uuid:54b0506d-49af-4245-9d40-d7d64902055e",
```

Comment:
Unfortunately for all these examples we probably shouldn't be just removing the Healthcare service resource, we should be translating it to the equivalent organisation resource.